app/controllers/billing_invoice.py

```diff
@@ -118,3 +122,71 @@ def post(self, project_id):
             message=f'Invoice for user with email {user.email} and project_id {project_id} sent successfully',
             data=dict(invoice=new_invoice_data)
         ), 200
+
+
+    @jwt_required
+    def get(self, project_id):
+
+        current_user_id = get_jwt_identity()
+        current_user_roles = get_jwt_claims()['roles']
+
+        billing_invoice_schema = BillingInvoiceSchema(many=True)
+        project = Project.get_by_id(project_id)
+
+        if not project:
+            return dict(status='fail', message='Project with project id {project_id} not found'), 404
+
+        
+        if not is_owner_or_admin(project, current_user_id, current_user_roles):
+                return dict(status='fail', message='Unauthorised'), 403
+
+        billing_invoice = BillingInvoice.find_all(project_id=project_id)
+
+        if not billing_invoice:
+            return dict(
+                status='fail',
+                message=f'billing invoice records not found'
+            ), 404
+
+        billing_invoice_data, errors = billing_invoice_schema.dumps(billing_invoice)
+
+        if errors:
+            return dict(status='fail', message=errors), 500
+
+        return dict(status='success', data=dict(
+            billing_invoice=json.loads(billing_invoice_data))), 200
```

Comment:
For readability purposes, this should be plural since it has multiple invoices.
```suggestion
            billing_invoices=json.loads(billing_invoice_data))), 200
```